pkg/imageset/updateinstaller.go

```diff
@@ -182,6 +182,12 @@ func (o *UpdateInstallerImageOptions) Run() (returnErr error) {
 		o.log.WithField("installerImage", installerImage).Info("installer image found")
 	}
 
+	installerImagePullPolicy := string(corev1.PullIfNotPresent)
+	if cd.Spec.Provisioning != nil && cd.Spec.Provisioning.InstallerImagePullPolicyOverride != "" {
+		installerImagePullPolicy = cd.Spec.Provisioning.InstallerImagePullPolicyOverride
+		o.log.WithField("installer image pull policy", installerImagePullPolicy).Info("installer image pull policy overridden")
```

Comment:
nit: I think keys for log fields are usually made without spaces so as to make the log line more easily parseable/greppable.